packages/modal-ui/src/lib/components/Modal.styles.ts

```diff
@@ -0,0 +1,373 @@
+export default `
+@import url('https://fonts.googleapis.com/css2?family=Manrope:wght@400;500;600&display=swap');
+
+#near-wallet-selector-modal {
+  --backdrop-bg: #26262630;
+  --heading-color: #262626;
+  --text-color: #676767;
+  --selected-wallet-bg: #A7A7A730;
+  --selected-wallet-bg-hover: transparent;
+  --wallet-option-border-color: #A7A7A730;
+  --content-bg: #FFFFFF;
+  --input-border-color-focus: #5F8AFA;
+  --box-shadow-color: #26262630;
+  --dismiss-button-bg-hover: #A7A7A730;
+  --dismiss-button-border-color-hover: inherit;
+  --confirm-button-color: #FFFFFF;
+  --confirm-button-bg: #5F8AFA;
+  --confirm-button-bg-hover: #5AA6FF;
+  --confirm-button-border-color: #5F8AFA;
+  --error: #DB5555;
+  --modal-close-button-color: #262626;
+}
+
+#near-wallet-selector-modal .dark-theme {
+  --backdrop-bg: #26262630;
+  --heading-color: #FFFFFF;
+  --text-color: #FFFFFF;
+  --selected-wallet-bg: #262626CC;
+  --selected-wallet-bg-hover: #262626CC;
+  --wallet-option-border-color: #A7A7A730;
+  --content-bg: #3F4246;
+  --input-border-color-focus: #5F8AFA;
+  --box-shadow-color: #26262630;
+  --dismiss-button-bg-hover: #262626CC;
+  --dismiss-button-border-color-hover: #262626;;
+  --confirm-button-color: #FFFFFF;
+  --confirm-button-bg: #5F8AFA;
+  --confirm-button-bg-hover: #5AA6FF;
+  --confirm-button-border-color: #5F8AFA;
+  --error: #DB5555;
+  --modal-close-button-color: #A7A7A7;
+}
+
+@media (prefers-color-scheme: dark) {
+  #wallet-selector-modal {
+    --backdrop-bg: #26262630;
+    --heading-color: #FFFFFF;
+    --text-color: #FFFFFF;
+    --selected-wallet-bg: #262626CC;
+    --selected-wallet-bg-hover: #262626CC;
+    --wallet-option-border-color: #A7A7A730;
+    --content-bg: #3F4246;
+    --input-border-color-focus: #5F8AFA;
+    --box-shadow-color: #26262630;
+    --dismiss-button-bg-hover: #262626CC;
+    --dismiss-button-border-color-hover: #262626;;
+    --confirm-button-color: #FFFFFF;
+    --confirm-button-bg: #5F8AFA;
+    --confirm-button-bg-hover: #5AA6FF;
+    --confirm-button-border-color: #5F8AFA;
+    --error: #DB5555;
+    --modal-close-button-color: #A7A7A7;
+  }
+}
+/**
+ * Modal Wrapper
+ */
+
+.modal-wrapper {
+  position: fixed;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  opacity: 0;
+  visibility: hidden;
+  //transition: visibility 0s linear 0.25s, opacity 0.25s 0s;
+  color: var(--wallet-selector-text-color, var(--text-color));
```

Comment:
Yes, it's true var takes n args as a fallback but in this case it would not work.

```css
color: var(--wallet-selector-text-color, #000); // applies fallback color
color: var(--wallet-selector-text-color, --text-color); // does not apply fallback color since --text-color is a css variable.
```

Reply:
I seriously need to read up on CSS! 😄